_includes/faq/auto-generate-unique-ids_v1.1.html

```diff
@@ -0,0 +1,21 @@
+To auto-generate unique row IDs, use the [`SERIAL`](serial.html) data type, which is an alias for [`INT`](int.html) with the `unique_rowid()` [function](functions-and-operators.html#id-generation-functions) as the [default value](default-value.html):
```

Comment:
^ @Amruta-Ranade, we should rework the guidance around auto-generating unique IDs.